invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/DocumentsSearch.js

```diff
@@ -119,7 +119,7 @@ export class DocumentsSearch extends Component {
         />
         <SortOrder
           values={config.SORT_ORDER}
-          defaultValue={config.SORT_ORDER[0]['value']}
+          defaultValue={config.SORT_ORDER['default_order']}
```

Comment:
you might want to use the `default_order` in all of the `SortOrder` components, in all search pages.